### ISSUE TITLE
Blue Sky Rendering Bug

### ISSUE THREAD

**Issue by AnotherCommander** [OoliteProject/oolite#84]
This is with latest trunk versions and does not manifest always but frequently enough to be notable and annoying: When diving inside the planet's atmosphere, sometimes the blue sky rendering shows polygon artifacts. In the image below you can see the problem from Edrebi's atmosphere in G5.
![ac_blueskyrenderbug](https://f.cloud.github.com/assets/623526/2108340/a7ad6ade-8fdd-11e3-9db2-23dfff5c554a.jpg)

Graphics details of my system below:

```
10:32:39.688 [rendering.opengl.version]: OpenGL renderer version: 2.1.0 ("2.1.0 - Build 8.15.10.2418"). Vendor: "Intel". Renderer: "Intel(R) HD Graphics".
10:32:39.688 [rendering.opengl.extensions]: OpenGL extensions (99):
GL_NV_primitive_restart, GL_ARB_texture_env_crossbar, GL_EXT_abgr, GL_ARB_shadow, GL_IBM_texture_mirrored_repeat, GL_EXT_separate_specular_color, 
GL_EXT_texture_edge_clamp, GL_EXT_blend_minmax, GL_EXT_draw_buffers2, GL_ARB_texture_env_add, GL_ARB_draw_elements_base_vertex, GL_ARB_sampler_objects, 
GL_EXT_framebuffer_blit, GL_EXT_transform_feedback, GL_ARB_texture_env_dot3, GL_EXT_framebuffer_object, GL_EXT_blend_subtract, 
GL_3DFX_texture_compression_FXT1, GL_NV_conditional_render, GL_ARB_vertex_array_bgra, GL_EXT_texture_snorm, GL_ARB_multitexture, GL_EXT_stencil_wrap, 
GL_ARB_texture_border_clamp, GL_ARB_framebuffer_sRGB, GL_ARB_texture_env_combine, GL_EXT_texture_env_add, GL_EXT_packed_pixels, 
GL_SGIS_texture_edge_clamp, GL_ARB_depth_clamp, GL_INTEL_performance_queries, GL_ARB_uniform_buffer_object, GL_ARB_depth_texture, GL_SGIS_texture_lod, 
GL_ARB_occlusion_query, GL_ARB_transpose_matrix, GL_NV_texgen_reflection, GL_EXT_stencil_two_side, GL_ARB_seamless_cube_map, 
GL_ARB_fragment_program_shadow, GL_ARB_shading_language_100, GL_ARB_texture_compression_rgtc, GL_EXT_gpu_program_parameters, 
GL_EXT_texture_shared_exponent, GL_EXT_rescale_normal, GL_ARB_map_buffer_range, GL_ARB_pixel_buffer_object, GL_EXT_secondary_color, GL_ARB_copy_buffer, 
GL_ARB_half_float_pixel, GL_EXT_blend_equation_separate, GL_EXT_draw_range_elements, GL_EXT_texture_lod_bias, GL_EXT_texture_filter_anisotropic, 
GL_ARB_half_float_vertex, GL_EXT_texture_swizzle, GL_SGIS_generate_mipmap, GL_ARB_texture_rectangle, GL_ARB_texture_non_power_of_two, GL_ARB_point_sprite, 
GL_EXT_packed_depth_stencil, GL_ARB_vertex_shader, GL_EXT_texture_compression_s3tc, GL_ARB_depth_buffer_float, GL_ARB_sync, GL_ARB_draw_instanced, 
GL_ARB_vertex_buffer_object, GL_WIN_swap_hint, GL_ARB_color_buffer_float, GL_ARB_shader_objects, GL_EXT_texture_array, GL_ATI_separate_stencil, 
GL_ARB_framebuffer_object, GL_EXT_packed_float, GL_ARB_compatibility, GL_EXT_texture_rectangle, GL_ARB_texture_rg, GL_ARB_fragment_shader, 
GL_EXT_fog_coord, GL_EXT_texture3D, GL_ARB_window_pos, GL_ARB_fragment_program, GL_ARB_vertex_program, GL_EXT_texture_env_combine, 
GL_ARB_point_parameters, GL_ARB_texture_cube_map, GL_EXT_bgra, GL_EXT_compiled_vertex_array, GL_ARB_vertex_array_object, GL_ARB_draw_buffers, 
GL_ARB_texture_compression, GL_EXT_multi_draw_arrays, GL_ARB_texture_float, GL_EXT_blend_func_separate, GL_EXT_texture_sRGB, GL_EXT_shadow_funcs, 
GL_EXT_clip_volume_hint, GL_EXT_blend_color, GL_NV_blend_square
10:32:39.863 [rendering.opengl.shader.support]: Shaders are supported.
```

**Comment by AnotherCommander** [comment]
I recently saw this bug occuring on a GTX960 with the latest version of the Cinematic Skies. Some investigation indicated that the nebula files were not 8-bit greyscale at all, but were rather 24-bit color. I converted all of them to 8-bit greyscale with transparent color set to black and the problem has been fixed. I will close this issue as soon as I can confirm the fix on my own computer, which should be possible in a few days.

Standard game nebula textures are fine and do not demonstrate any kind of problem.
